src/lib/ProjectCategory.svelte

```diff
@@ -48,23 +48,30 @@
   }
 
   function setAllProjects() {
-    categories = categories.map((category) => ({ ...category, active: false }));
-    dispatch('categorySelected', '');
-  }
+  categories = categories.map((category) => ({
+    ...category,
+    active: category.id === 0, // Set "All Projects" active when clicked
+  }));
+
+  dispatch('categorySelected', '');
+}
+
```

Comment:
@HawaKallon any comments on this section??
everything else looks ok _(just minor class sorting & tweaks)_